Dockerfile.template.erb

```
@@ -68,7 +68,9 @@ RUN choco install -y ruby --version 3.2.4.1 --params "'/InstallDir:C:\ruby32'" \
 && choco install -y msys2 --version 20240507.0.0 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby32\msys64'"
 RUN refreshenv \
 && ridk install 3 \
-&& echo gem: --no-document >> C:\ProgramData\gemrc \
+&& type "c:\ProgramData\gemrc" \
+&& c:\ruby32\bin\ruby -e "require 'yaml'; File.open('c:/ProgramData/gemrc') do |file| yaml = YAML.safe_load(file); yaml['install'] = '--no-document'; yaml['update'] = '--no-document'; File.open('c:/ProgramData/gemrc', 'w+') do |output| output.puts(YAML.dump(yaml)); end; end" \
```

Do we need double block?
How about simply doing the following?

```suggestion
&& c:\ruby32\bin\ruby -r yaml -e "path = 'c:/ProgramData/gemrc'; yaml = YAML.safe_load(File.read(path)); yaml['install'] = '--no-document'; yaml['update'] = '--no-document'; File.write(path, YAML.dump(yaml))" \
```

---

Looks good.